lkvt.go

```diff
@@ -261,23 +319,43 @@ func (conf *config) execute(c int, ran []uint32, wg *sync.WaitGroup) {
 			randVal:   toByteArray(ran[i]),
```

Comment:
This function and `remainder()` are essentially duplicates.   If you want to get an exact operation count per 'thread' then just create an integer that's the sum of `n = ((x / y) + (x % y))` and use `n` as the for loop's test value.  `remainder()` should be removed.